academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md

```diff
@@ -2,27 +2,1410 @@
+    
+    
+    //////////////////////////////////////////////////////////
+    // Output files by entity type. Used for debug purposes.
+    //////////////////////////////////////////////////////////
+    IF @outputFileByEntityType THEN
+        OUTPUT @paperJson
+        TO @Out_OutputPath + "/papers/{PartitionNumber}.json"
+        USING Outputters.Tsv(quoting : false);
+    
+        OUTPUT @affiliationJson
+        TO @Out_OutputPath + "/affiliations/{PartitionNumber}.json"
+        USING Outputters.Tsv(quoting : false);
+    
+        OUTPUT @authorJson
+        TO @Out_OutputPath + "/authors/{PartitionNumber}.json"
+        USING Outputters.Tsv(quoting : false);
+    
+        OUTPUT @conferenceInstanceJson
+        TO @Out_OutputPath + "/conferenceInstances/{PartitionNumber}.json"
+        USING Outputters.Tsv(quoting : false);
+    
+        OUTPUT @conferenceSeriesJson
+        TO @Out_OutputPath + "/conferenceSeries/{PartitionNumber}.json"
+        USING Outputters.Tsv(quoting : false);
+    
+        OUTPUT @fieldOfStudyJson
+        TO @Out_OutputPath + "/fieldsOfStudy/{PartitionNumber}.json"
+        USING Outputters.Tsv(quoting : false);
+    
+        OUTPUT @journalJson
+        TO @Out_OutputPath + "/journals/{PartitionNumber}.json"
+        USING Outputters.Tsv(quoting : false);
+    END;
+    ```
+
+1. Replace placeholder values in the script using the table below
+
+   |Value  |Description  | Example |
+   |---------|---------|------|
+   |**`@In_MagBlobAccount`** | The name of your Azure Storage account containing the Microsoft Academic Graph data set. | **mymagstore**
+   |**`@In_MagBlobContainer`** | The container name in your Azure Storage account containing the Microsoft Academic graph data set, usually in the form of **mag-YYYY-MM-DD**. | **mag-2020-02-07**
+   |**`@Out_OutputPath string`** | The entity output wasb path to where you'd like the formatted entities documents to go. | **wasb://makessubgraph@mymakesstore/2020-02-07/microsoft/entities**  |
+   |**`@Param_UseSubgraphForInstitution`** | Generates a subgraph containing data related to the specified institution. | **microsoft**|
+
+    > [!TIP]
+    > This tutorial uses Microsoft as an organization by default. You can target any organization by finding its NormalizedName in the Microsoft Academic Graph and then changing the value of the affiliationNormalizedNameFilter variable to said name.
+    >
+    > Please note that doing this may impact time estimates for script execution and index generation later on in this tutorial.
+
+5. Provide a **Job name**, change **AUs** to 50, and select **Submit**
+
+   ![Submit GenerateAzureSearchData job](media/how-to-create-index-create-job.png)
+
+The job should finish successfully in about 50 minutes
+
 ## Build index using MAKES command line tool
 
+Now that the data has been generated to build your index, we need to create some resources that the index will use and build the index using the kesm.exe tool.
```

Comment:
Now that the JSON data has been generated to build the custom index, we use the KESM tool to execute the build.